client/state/reader/follows/reducer.js

```diff
@@ -64,28 +66,48 @@ function updatePostSubscription( state, { payload, type } ) {
 	};
 }
 
-/**
- * Tracks all known list objects, indexed by list ID.
- *
- * @param  {Array} state  Current state
- * @param  {Object} action Action payload
- * @return {Array}        Updated state
- */
 export const items = createReducer( {}, {
-	[ READER_FOLLOW ]: ( state, action ) => {
+	[ READER_RECORD_FOLLOW ]: ( state, action ) => {
 		const urlKey = prepareComparableUrl( action.payload.url );
 		return {
 			...state,
 			[ urlKey ]: merge( {}, state[ urlKey ], { is_following: true } ),
 		};
 	},
-	[ READER_UNFOLLOW ]: ( state, action ) => {
+	[ READER_RECORD_UNFOLLOW ]: ( state, action ) => {
 		const urlKey = prepareComparableUrl( action.payload.url );
 		return {
 			...state,
 			[ urlKey ]: merge( {}, state[ urlKey ], { is_following: false } ),
 		};
 	},
+	[ READER_FOLLOW ]: ( state, action ) => {
+		const urlKey = prepareComparableUrl( action.payload.feedUrl );
+		return {
+			...state,
+			[ urlKey ]: merge(
+				{ feed_URL: action.payload.feedUrl },
```

Comment:
I think I wrote this before we added feed_URL to the subscription object (it's just plain URL from the API). 

It does raise a related issue. If you follow a URL and we determine in the API that that URL isn't the actual feed URL (say you follow `https://example.com`, but the feed url for it is actually `https://example.com/feed`, we should handle that... Need to verify that that's a thing that actually happens though. Only real case is direct URL follows and that may end up going a different path through redux.